src/components/Header/header.less

```diff
@@ -107,18 +108,34 @@
   }
 }
 
-.headerContainer__account {
-  display: inline-block;
+.headerContainer {
+
+  &__network {
+    display: inline-block;
```

Comment:
I was not able to find an element that will match `.headerContainer__network` selector, so probably we do not need `display` property here